### ISSUE TITLE
Details of training on the KIT dataset

### ISSUE THREAD

**Issue by lunarlabour** [neu-vi/OmniControl#16]
Hello, I have been trying to train the model on the KIT dataset, but I failed to reproduce the result in the paper. For the train, I simply changed the dataset argument in the provided script and nothing else. The Loss term was still as high as 0.3 at the end of the training procedure and is abnormal compared to the ML3D result. 
Is there something I'm doing wrong? Could you provide some details of training on the KIT dataset like train script or the log? It would be very helpful and appreciated.

**Comment by ymingxie** [comment]
Hi sorry for the late response. The unit for KIL is centimeter instead of meter like in HumanML3D. Maybe that's why the loss is higher than the loss trained on HumanML3D. 
Are the evaluation results also abnormal?

**Comment by lunarlabour** [comment]
Thanks for the reply! 

The evaluation results are also not as expected. 

The loss at the beginning of my training process was around 1.6, and it went down to 0.3 by the end (400K steps). Such small magnitude difference may imply the model is not fully trained. 

Here are my training log and evaluation results.

![log](https://github.com/neu-vi/OmniControl/assets/71216210/b4369c5d-081a-4be3-98b5-030d408fb5af)

![val](https://github.com/neu-vi/OmniControl/assets/71216210/486724a2-bf2e-4d1b-bd21-1214546e27e2)

**Comment by ymingxie** [comment]
Thank you for the information. It seems you might be loading the pretrained weights on HumanML3D. Could you please try to load [kit_pretrain_model](https://drive.google.com/file/d/1c8SA8HYJCdEBiLYJQn8kdYQJwv6x3tv_/view?usp=sharing)? This checkpoint is adopted from [MDM](https://github.com/GuyTevet/motion-diffusion-model?tab=readme-ov-file#3-download-the-pretrained-models).
